src/components/MagicAnswer.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+export default function MagicAnswer({ answers }) {
+  const [answer, setAnswer] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const randomIndex = () => {
+    const random = Math.floor(Math.random() * answers.length);
+    setAnswer(answers[random]);
+    setIsLoading(true);
```

Comment:
There's no need to add a `isLoading` state into this, because there won't be any delay since this isn't asynchronous.

Reply:
Plus you have a bug here, when a user clicks on the button, this will always be true, and it won't show the answer.
instead you should use the not operator `!` to reverse the value.
e.g: `setIsLoading(!isLoading)`